stratosphere/ams_mitm/source/utils.cpp

```diff
@@ -665,3 +667,91 @@ Result Utils::GetSettingsItemBooleanValue(const char *name, const char *key, boo
 void Utils::RebootToFatalError(AtmosphereFatalErrorContext *ctx) {
     BpcRebootManager::RebootForFatalError(ctx);
 }
+
+void Utils::CreateBlankProdinfoIfNeeded() {
+    const bool p = Utils::GetCAL0BackupBufferToBlank();
```

Comment:
Have this actually return a buffer instead of a bool. This will remove ugly p ? x : y from the rest of code.